lib/parameter_substitution/expression.rb

```diff
@@ -77,6 +91,28 @@ def validate_expressions
       end
     end
 
+    def unknown_method_messages
+      unknown_parameter_methods.map_compact do |param, methods|
+        method_string = "method#{methods.size > 1 ? 's' : ''}"
+        if !(methods.empty? || unknown_parameters.include?(param))
+          "Unknown #{method_string} #{@context.formatted_arg_list(methods)} used for on parameter '#{param}'"
+        end
+      end
+    end
+
+    def unknown_parameter_messages
+      unknown_parameters.map do |param|
+        unknown_methods_for_param = unknown_parameter_methods[param]
+        method_string = "method#{unknown_methods_for_param.size > 1 ? 's' : ''}"
```

Comment:
Same comment(s) about pluralization.

Reply:
Dried up and pushed it. I commented on this above and pushed it up.